.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "linux"]
+	path = linux
+	url = git://github.com/torvalds/linux.git
```

Comment:
I'd rather prefer to have a submodule here, especially such a big one. For local checks I'll use pre-existing my local linux tree anyway.